docs/api.md

```diff
@@ -616,8 +617,6 @@ var sync = PouchDB.sync('mydb', 'http://localhost:5984/mydb', {live: true})
     // handle change
   }).on('complete', function (info) {
     // handle complete
-  }).on('uptodate', function (info) {
-    // handle up-to-date
   }).on('error', function (err) {
```

Comment:
This wasnt an exhaustive list so I didnt thought that was a little confusing, change / complete and error are the primary events. I want to have a more explanatory documentation but that needs a little prework to make sure it aligns with the guide / api docs